src/main/java/com/encore/playground/domain/notice/entity/Notice.java

```diff
@@ -0,0 +1,36 @@
+package com.encore.playground.domain.notice.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.ColumnDefault;
+
+import java.time.LocalDateTime;
+
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Builder
+@Entity
+@Table(name="notice")
+
+public class Notice {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long noticeId;
+    @Column(nullable = false)
+    private String title;
+    @Column(nullable = false)
+    private String author;
+    @Column(nullable = false)
+    private String contents;
+    @Column(nullable = false)
+    private LocalDateTime uploadTime;
+    @Column(nullable = false)
+    @ColumnDefault("0")
+    private int viewCount;
```

Comment:
#9 의 Feed entity에서 선언했던 변수명이 있는데요

```
public class Feed {
    @Id
    @Column(nullable = false)
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long feedNo; // 글번호
    @Column(nullable = false, length = 20)
    private String userid; // 작성자
    @Column(nullable = false)
    private LocalDateTime uploadTime; // 작성일자
    @Column(nullable = false)
    @ColumnDefault("0")
    private int likeCount; // 좋아요 수
    @Column(nullable = false)
    @ColumnDefault("0")
    private int commentCount; // 댓글 수
    @Column(nullable = false)
    @ColumnDefault("0")
    private int viewCount; // 조회 수
    @Column(nullable = false, length = 1000)
    private String article; // 글 내용
}
```

그래서 다음과 같이 수정을 요청합니다.
```suggestion
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long noticeNo;
    @Column(nullable = false)
    private String title;
    @Column(nullable = false)
    private String userid;
    @Column(nullable = false)
    private String article;
    @Column(nullable = false)
    private LocalDateTime uploadTime;
    @Column(nullable = false)
    @ColumnDefault("0")
    private int viewCount;
```

Reply:
넵 의논 후 결정하도록 합시다.

Reply:
네 그럼 변수명은 추후에 쭉 보는걸로

Reply:
섹시해요 리차드